athena-sqlserver/src/main/resources/log4j2.xml

```diff
@@ -10,6 +10,7 @@
     <Loggers>
         <Logger name="com.amazonaws.athena.connector.lambda" level="${env:ATHENA_FEDERATION_SDK_LOG_LEVEL:-warn}" />
         <Logger name="com.amazonaws.athena.connectors.jdbc" level="${env:ATHENA_FEDERATION_SDK_LOG_LEVEL:-warn}" />
+        <Logger name="com.amazonaws.athena.connectors.sqlserver" level="${env:ATHENA_FEDERATION_SDK_LOG_LEVEL:-debug}" />
```

Comment:
Did you intend to set the default level for this logger to `debug`?
If so, why is this `debug` instead of `warn` like the others?

Reply:
agree with henry